src/mscorlib/src/System/IO/Stream.cs

```diff
@@ -871,6 +821,23 @@ internal static void BlockingEndWrite(IAsyncResult asyncResult)
         {
             SynchronousAsyncResult.EndWrite(asyncResult);
         }
+        
+        internal void ValidateCopyToArguments(Stream destination, int bufferSize)
```

Comment:
I'm guessing we don't really care about code contracts these days, but should this method have `[ContractArgumentValidator]`?